libkbfs/backpressure_disk_limiter.go

```diff
@@ -1075,19 +1075,27 @@ type backpressureDiskLimiterStatus struct {
 }
 
 func (bdl *backpressureDiskLimiter) getStatus(
-	chargedTo keybase1.UserOrTeamID) interface{} {
+	ctx context.Context, chargedTo keybase1.UserOrTeamID) interface{} {
 	bdl.lock.RLock()
 	defer bdl.lock.RUnlock()
 
 	currentDelay := bdl.getDelayLocked(
 		context.Background(), time.Now(), chargedTo)
 
+	jStatus := bdl.journalTracker.getStatus(chargedTo)
+	// If we haven't updated the quota limit yet, update it now.
+	if jStatus.QuotaStatus.QuotaBytes == math.MaxInt64 {
```

Comment:
I'm wondering if we should use an error status instead. But I don't feel strongly about it.